src/components/Books.js

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getDataFromAPi } from '../redux/books/books';
 import Book from './Book';
 import Input from './Input';
+// import { getDataFromAPi } from './..redux/books/books';
```

Comment:
- [ ] **Optional:** Kindly remove commented code from your codebase. It is considered bad practice to commit commented code. And it also makes code hard to read.

Reply:
Hello @brytebee. Thanks for the code review. I will implement it and then merge it. Thanks again